repo_health/index/static/client/src/components/chart/chart.component.js

```diff
@@ -0,0 +1,45 @@
+/*
+* chart.component.js - (C) Copyright - 2017
+* This software is copyrighted to contributors listed in CONTRIBUTIONS.md.
+*
+* SPDX-License-Identifier: MIT
+*
+* Author(s) of this file:
+* J.Harding
+*
+* Create chart component.
+*/
+
+import template from './chart.template';
+
+const chartComponent = {
+  template,
+  bindings: {
+    'metric': '=',
+    'chart': '='
+  },
+  controller: class chartComponentController {
+    options = null;
+    hasData = true;
+    titleText = '';
+    constructor() {
```

Comment:
You don't need this constructor if nothing is being injected into this controller.